container/test/rbe-debian8.yaml

```diff
@@ -1,9 +1,21 @@
 schemaVersion: "1.0.0"
 
 commandTests:
```

Comment:
please add a couple more tests to verify that the base packages are installed properly (e.g., check git, wget, etc)

Reply:
Done. Tests added to verify all language interpreters/compilers and tools.